### ISSUE TITLE
Uploading metadata

### ISSUE THREAD

**Issue by peflanag** [achtman-lab/GrapeTree#92]
Hi,

I use the website https://achtman-lab.github.io/GrapeTree/MSTree_holder.html to visualise trees and for the time being have been entering metadata manually. Is there a way to upload the metadata from an excel file or to copy and past it in? I have vast amounts.

Cheers,

Peter

**Comment by zheminzhou** [comment]
Yes, you can upload a tab-delimited file describing metadata into the GrapeTree (via "load files"). 
The first row of the file needs to be the name of metadata fields, and the other rows describe the metadata content of every genome. 
The only requirement is that a column named "ID" needs to be consistent with the tip names in the tree. 

You can find examples of the metadata files under 
https://github.com/achtman-lab/GrapeTree/blob/master/examples/simulated_data.metadata.txt

This file is used to annotate the tree:
https://github.com/achtman-lab/GrapeTree/blob/master/examples/simulated_data.global.nwk

**Comment by peflanag** [comment]
Hey @zheminzhou thats great thanks so much for the info!

**Comment by happykhan** [comment]
Closing this as answered. GrapeTree accepts tab-delimited metadata through **Load Files**; the table must contain an `ID` column matching the tree tip names. The repository example remains at `examples/simulated_data.metadata.txt`.
